cmd/main.go

```diff
@@ -110,7 +110,6 @@ func main() {
 
 	loggingSecrets := loggingcredentials.Reconciler{
 		Client: mgr.GetClient(),
-		Scheme: mgr.GetScheme(),
```

Comment:
You are right, we do not need to pass Scheme along. We could simplify this.

Reply:
It has been done in commit 8d26017a1c852da239f0362797a5ec341467b1e3